pkg/sideinput/examples/simple-sideinput/pipeline.yaml

```diff
@@ -0,0 +1,41 @@
+apiVersion: numaflow.numaproj.io/v1alpha1
+kind: Pipeline
+metadata:
+  name: my-pipeline
+spec:
+  sideInputs:
+    - name: myticker
+      container:
+        image: "quay.io/numaio/numaflow-go/sideinput:v0.5.0"
```

Comment:
Is this the real image? use something like `udf-sideinput-example` as described in the Dockerfile?